libnd4j/buildnativeoperations.sh

```diff
@@ -265,12 +265,18 @@ case "$OS" in
     windows*)
     # Do something under Windows NT platform
     if [ "$CHIP" == "cuda" ]; then
-        export CMAKE_COMMAND="cmake -G \"NMake Makefiles\""
-        export MAKE_COMMAND="nmake"
-        PARALLEL="false"
+        export CMAKE_COMMAND="cmake -G \"Ninja\""
+        export MAKE_COMMAND="ninja"
+        export CC="cl.exe"
+        export CXX="cl.exe"
+        PARALLEL="true"
     else
         export CMAKE_COMMAND="cmake -G \"MSYS Makefiles\""
         export MAKE_COMMAND="make"
+
+        # Sam, do we really need this?
```

Comment:
Probably not... BTW. why not use ninja for everything, not just Windows CUDA?

Reply:
I was solving 1 particular problem: CUDA compilation time on Windows.
For other platforms we have regular make, which works just fine afaik. Only Windows+CUDA had sequential builds.

Reply:
p.s. Ninja is available as package for MinGW.

Reply:
Meh, I guess we can switch all to ninja gradually :) @sshepel ?

Reply:
Well, if build script code will be the same for all platforms - that would be good definitely. We just need to make sure there's no issues with Ninja availability on other platforms: Linux, macOS.